### PR TITLE
Line Plot Amendments

### PR DESCRIPTION
**Pull request by lilyclements**

### CODE REVIEW

File: instat/dlgPlot.vb [hunk 1]
```diff
@@ -132,6 +123,15 @@ Public Class dlgPlot
         TestOkEnabled()
     End Sub
 
+    Public Sub SetRCodeForControls(bReset As Boolean)
```
Comment (lilyclements) [inline instat/dlgPlot.vb:126]: Made this sub Public since it has a sdg.

File: instat/dlgPlot.vb [hunk 2]
```diff
@@ -193,19 +190,11 @@ Public Class dlgPlot
         TestOkEnabled()
     End Sub
 
-    Private Sub AllControl_ControlContentsChanged() Handles ucrReceiverX.ControlContentsChanged, ucrVariablesAsFactorForLinePlot.ControlContentsChanged, ucrSave.ControlContentsChanged
-        TestOkEnabled()
-    End Sub
-
-    Private Sub ucrChkPoints_CheckedChanged(ucrChangedControl As ucrCore)
```
Comment (lilyclements) [inline instat/dlgPlot.vb:200]: Removed empty subs

File: instat/dlgPlot.designer.vb [hunk 3]
```diff
@@ -66,7 +66,7 @@ Partial Class dlgPlot
         '
         'cmdLineOptions
         '
-        Me.cmdLineOptions.Location = New System.Drawing.Point(10, 228)
+        Me.cmdLineOptions.Location = New System.Drawing.Point(10, 198)
```
Comment (lilyclements) [inline instat/dlgPlot.designer.vb]: Switched the two command buttons around. Other dialogs tend to have
Dialog-name Options
Plot Options

This dialog had it the other way

File: instat/dlgPlot.vb [hunk 4]
```diff
@@ -50,24 +49,24 @@ Public Class dlgPlot
         ucrLinePlotSelector.SetParameter(New RParameter("data"))
         ucrLinePlotSelector.SetParameterIsrfunction()
 
+        ucrReceiverX.SetParameter(New RParameter("x"))
```
Comment (lilyclements) [inline instat/dlgPlot.vb:52]: New Parameter at the start of their code

File: instat/dlgPlot.vb [hunk 5]
```diff
@@ -38,7 +38,6 @@ Public Class dlgPlot
         TestOkEnabled()
     End Sub
 
-
```
Comment (lilyclements) [inline instat/dlgPlot.vb]: Removed double space

File: instat/dlgPlot.vb [hunk 6]
```diff
@@ -113,8 +113,8 @@ Public Class dlgPlot
         clsRaesFunction.AddParameter("x", Chr(34) & Chr(34))
         clsRaesFunction.AddParameter("y", Chr(34) & Chr(34))
 
-        clsRgeomlineplotFunction.SetRCommand("geom_line")
         clsRgeomlineplotFunction.SetPackageName("ggplot2")
+        clsRgeomlineplotFunction.SetRCommand("geom_line")
```
Comment (lilyclements) [inline instat/dlgPlot.vb:117]: SetPackageName before SetRCommand

**Comment by lilyclements** [comment]
@maxwellfundi This is ready for review

**Comment by maxwellfundi** [comment]
@lilyclements why do you have so many files changed here?

**Comment by lilyclements** [comment]
@maxwellfundi It's because this branch was more updated than your one so it's merging in all those updates. All these file changes won't come up on your branch when you've merged it. People have merged in ones with more file changes than this

**Comment by maxwellfundi** [comment]
Hehehe okay.

**Pull request merged**